### ISSUE TITLE
Status of provisional meta team

### ISSUE THREAD

**Issue by asdofindia** [MozillaIndia/meta#2]
It's been 40+ days after the meetup and it's about time to think if all is good.

Apart from this repository and two conference calls with spare attendance, I'm not sure the provisional meta team has been able to produce any tangible output.

Is it time to reconsider membership of provisional meta team? Should we recruit more active members? 

Should we drop the entire idea of provisional meta team? Should we directly form a meta team the way other teams are forming?

Should we forget about the meetup and restructure and let things continue as they are?

**Comment by gurumukhi** [comment]
The answer of last question is definitely clear - There is no way that we should forget about the meetup and restructure and let things continue as they are!

I suggest:
- To confirm with provisional meta team's members if they can put it enough time & efforts to push things. Someone from the team itself should do this check.
- We should open up to recruit additional members who could be more promising towards success of this idea. We can consider the active members of currently active functional & focus groups to include in the provisional meta team. The members of the team should decide how to go ahead with this.

Another point from my side:
I think we should focus on the first goal of provisional team (to Support creation of functional and focus teams ref-https://discourse.mozilla-community.org/t/getting-to-work-as-provisional-meta-team/11078) separately irrespective of the success/progress of the team, because as other teams the meta team might also take time to get started for practical reasons. The reason/intention behind this statement is that this goal looks very important to me in the bigger picture of Mozilla India restructure. I don't have any strong suggestion about how to get this goal met sooner, but I think we can consider brainstorming this goal separately. 

^ These are my thoughts considering all practical reasons (which I could think of) in mind, feel free to pitch in what you think! #Peace :-D

**Comment by KotechaVishwa** [comment]
Yes,  I completely agree . And the teams must be formed sooner as possible , so that every contributor gets a fixed domain to work on.

**Comment by kaustavdm** [comment]
Agree with @asdofindia, @gurumukhi and @KotechaVishwa.

I see the action items as:
- Solidify a criteria for the meta team.
- Open up a call for participation and get in applications.
- Get an asynchronous process set up. Meetings clearly don't scale.

**Comment by RCMainak** [comment]
Would anyone like to catch me up about what's going on here. Sorry for not keeping up with the community lol.

Also I would appreciate if someone could add me to the Mozilla India Telegram group. Thanks in advance :smiley:

**Comment by kaustavdm** [comment]
X-linking discourse/mailing list discussion: http://discourse.mozilla-community.org/t/status-of-provisional-meta-team/11467/3
